pkg/controller/keepalivedgroup/keepalivedgroup_controller.go

```diff
@@ -236,8 +235,26 @@ func getNamespaceNameKey(obj metav1.Object) string {
 	return obj.GetNamespace() + "/" + obj.GetName()
 }
 
+func isInList(sid int, ids []int) bool {
```

Comment:
Why is this function implemented, it is only called twice - Is this just to save space?

Reply:
It is to improve readability. And twice should be enough to create a function, no?

Reply:
with regards to the `getNamespaceNameKey` function, this is part of the operator-utils library now. So please call that one.
it is called `apis.GetKeyShort()`.

Reply:
Sorry, which line are you reffering to?

Reply:
sorry I might have muddied the waters here, I was referring to any appearance of the `getNamespaceNameKey` function.

Reply:
@danielkucera After an internal discussion we've decided it would be better to leave the function out for now. I understand it is best practice we deemed it is too much of a change

Reply:
function removed